content/x-ray/microservices.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Deploy Example Microservices"
+date: 2018-11-177T08:30:11-07:00
+weight: 12
+draft: false
+---
+
+We now have the foundation in place to deploy microservices, which are instrumented with [X-Ray SDKs](https://docs.aws.amazon.com/xray/index.html#lang/en_us), to the EKS cluster.
+
+In this step, we are going to deploy example [front-end](https://github.com/rnzsgh/eks-workshop-x-ray-sample-front) and [back-end](https://github.com/rnzsgh/eks-workshop-x-ray-sample-back) microservices to the cluster. The example services are already instrumented using the [X-Ray SDK for Go](https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-go.html). Currently, X-Ray has SDKs for Go, Python, Node.js, Ruby, .NET and Java.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/rnzsgh/eks-workshop-x-ray-sample-back/master/x-ray-sample-back-k8s.yml
+
+kubectl apply -f https://raw.githubusercontent.com/rnzsgh/eks-workshop-x-ray-sample-front/master/x-ray-sample-front-k8s.yml
+```
+
+To review the status of the deployments, you can run:
+
+```
+kubectl describe deployments x-ray-sample-front-k8s x-ray-sample-back-k8s
+```
+
+For the status of the services, run the following command:
+
+```
+kubectl describe services x-ray-sample-front-k8s x-ray-sample-back-k8s
```

Comment:
What is the expected output? What is the user looking for to know that this DaemonSet is working?